src/build.rs

```diff
@@ -422,66 +466,149 @@ impl BuildQueue {
         // However, if Cargo doesn't run a separate thread, then we'll just wait
         // forever. Therefore, we spawn an extra thread here to be safe.
         let handle = thread::spawn(move || {
-            let mut flags = "-Zunstable-options -Zsave-analysis --error-format=json \
-                             -Zcontinue-parse-after-error".to_owned();
+            #[derive(Debug)]
+            struct CargoOptions {
```

Comment:
Cargo in its API expects references to values that will live at least as long as `compile_with_exec`, so I just wanted to logically bundle them together